src/mediapipe_internal/mediapipe_utils.cpp

```diff
@@ -40,7 +40,17 @@ const std::string MP_IMAGE_PREFIX{"IMAGE"};
 
 const std::string EMPTY_STREAM_NAME{""};
 
-std::pair<std::string, mediapipe_packet_type_enum> getStreamNamePair(const std::string& streamFullName) {
+std::pair<std::string, mediapipe_packet_type_enum> getStreamNamePair(const std::string& streamFullName, const MediaPipeStreamType& streamType) {
```

Comment:
```suggestion
std::pair<std::string, mediapipe_packet_type_enum> getStreamNamePair(const std::string& streamFullName, MediaPipeStreamType streamType) {
```
No need to pass integers via const reference